### PR TITLE
Added comments to the dictionary defining the symbols used in it.

### PR DESCRIPTION
**Pull request by gs-chaitanya**
Google Code In Task 2019 by Daedalus

### CODE REVIEW

File: apertium-eng-cat.eng-cat.dix [hunk 1]
```diff
@@ -11,99 +11,120 @@
...
+    <sdef n="cog" c="Cognomen"/>
+    <sdef n="org"  c="Organisation"/> 
+<!--Number-->
+    <sdef n="sg"    c="Singular"/> 
+    <sdef n="pl"  c="Plural"/> 
+    <sdef n="ND"  c="Number to be determined"/> 
+    <sdef n="sp"  c="Singular or plural	"/> 
+<!--case-->
+    <sdef n="gen"  c="Genitive"/> 
+<!--Voice-->
```
Comment (jonorthwash) [inline apertium-eng-cat.eng-cat.dix:92]: Why the empty ~category?

File: apertium-eng-cat.eng-cat.dix [hunk 2]
```diff
@@ -11,99 +11,120 @@
...
+    <sdef n="dem"  c="Demonstrative"/> 
+    <sdef n="def"  c="Definite"/> 
+    <sdef n="detnt"  c="Neuter determiner"/> 
+    <sdef n="atn" c="Atónico"/> 
+    <sdef n="pos"  c="Possessive"/> 
+    <sdef n="ref"  c="Reflexive"/>
+    <sdef n="subj"  c="Subject"/> 
+    <sdef n="obj"  c="Object	"/> 
+    <sdef n="ord"  c="Ordinal"/>  
+<!--Transivity-->
```
Comment (jonorthwash) [inline apertium-eng-cat.eng-cat.dix:76]: Why the empty ~category?

**Review by jonorthwash — CHANGES_REQUESTED** [review]
There's an error (at least one).  Have you tried to validate the dictionary?

File: apertium-eng-cat.eng-cat.dix (hunk 2 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:76]: Sorry,
Will change immediately.

File: apertium-eng-cat.eng-cat.dix [hunk 3]
```diff
@@ -11,99 +11,116 @@
...
+    <sdef n="sint"  c="Synthetic"/> 
+<!--Others-->
+    <sdef n="web"  c="Links and Emails"/> 
+    <sdef n="percent"  c="Percentage"/> 
+    <sdef n="mon"  c="Money"/> 
+    <sdef n="maj"  c="majuscule"/> 
+    <sdef n="min"  c="Miniscule"/> 
+    <sdef n="file"  c="Filename"/>
+    <sdef n="adjant"  c=""/> 
     <sdef n="pron" />
```
Comment (gs-chaitanya) [inline apertium-eng-cat.eng-cat.dix:121]: This tag was missing.
It is perfect now.

**Review by gs-chaitanya — COMMENTED** [review]
The dictionary is now perfect and is validated.

File: apertium-eng-cat.eng-cat.dix [hunk 4]
```diff
@@ -11,99 +11,116 @@
...
-    <sdef n="unc" /> 
-    <sdef n="sep" /> 
+  <!--Part-of-Speech Categories-->
+    <sdef n="n" c="Noun"/> 
+    <sdef n="np"  c="Proper noun"/> 
+    <sdef n="adj"  c="Adjective"/> 
+    <sdef n="adv" c="Adverb"/> 
+    <sdef n="preadv"  c="Pre-adverb"/> 
+    <sdef n="pr" c="Preposition"/> 
+    <sdef n="prn"  c="pronoun"/>
```
Comment (mr-martian) [inline apertium-eng-cat.eng-cat.dix:21]: should be capitalized to be consistent with the rest of the comments

File: apertium-eng-cat.eng-cat.dix [hunk 5]
```diff
@@ -11,99 +11,116 @@
...
+<!--Punctuation-->
+    <sdef n="sent"  c="Sentence-ending punctuation"/> 
+    <sdef n="cm"  c="Comma punctuation"/> 
+    <sdef n="lpar"  c="Left parenthesis"/> 
+    <sdef n="rpar"  c="Right parenthesis"/> 
+    <sdef n="lquest"  c="Left question/exclamation mark"/> 
+    <sdef n="guio"  c="Hyphen"/> 
+    <sdef n="apos"  c="Apostrophe"/> 
+<!--Gender-->
+    <sdef n="f" c="	Feminine"/>
```
Comment (mr-martian) [inline apertium-eng-cat.eng-cat.dix:41]: shouldn't have extra space in definition

File: apertium-eng-cat.eng-cat.dix [hunk 6]
```diff
@@ -11,99 +11,116 @@
...
+    <sdef n="itg"  c="Interrogative"/> 
+    <sdef n="det"  c="Determiner"/> 
+    <sdef n="dem"  c="Demonstrative"/> 
+    <sdef n="def"  c="Definite"/> 
+    <sdef n="detnt"  c="Neuter determiner"/> 
+    <sdef n="atn" c="Atónico"/> 
+    <sdef n="pos"  c="Possessive"/> 
+    <sdef n="ref"  c="Reflexive"/>
+    <sdef n="subj"  c="Subject"/> 
+    <sdef n="obj"  c="Object	"/>
```
Comment (mr-martian) [inline apertium-eng-cat.eng-cat.dix:72]: trailing space

File: apertium-eng-cat.eng-cat.dix [hunk 7]
```diff
@@ -11,99 +11,116 @@
...
+    <sdef n="top"  c="Toponym"/> 
+    <sdef n="ant"  c="Anthroponym"/>
+    <sdef n="al"  c="Altres"/> 
+    <sdef n="cog" c="Cognomen"/>
+    <sdef n="org"  c="Organisation"/> 
+<!--Number-->
+    <sdef n="sg"    c="Singular"/> 
+    <sdef n="pl"  c="Plural"/> 
+    <sdef n="ND"  c="Number to be determined"/> 
+    <sdef n="sp"  c="Singular or plural	"/>
```
Comment (mr-martian) [inline apertium-eng-cat.eng-cat.dix:86]: trailing space

**Review by mr-martian — CHANGES_REQUESTED** [review]
looks like just a couple more things

File: apertium-eng-cat.eng-cat.dix (hunk 3 above)
Comment (jonorthwash) [reply apertium-eng-cat.eng-cat.dix:121]: Why did you add it under "Others"?

File: apertium-eng-cat.eng-cat.dix [hunk 8]
```diff
@@ -11,99 +11,116 @@
...
+    <sdef n="sep"  c="Separable verb"/> 
+<!--Proper Nouns-->
+    <sdef n="top"  c="Toponym"/> 
+    <sdef n="ant"  c="Anthroponym"/>
+    <sdef n="al"  c="Altres"/> 
+    <sdef n="cog" c="Cognomen"/>
+    <sdef n="org"  c="Organisation"/> 
+<!--Number-->
+    <sdef n="sg"    c="Singular"/> 
+    <sdef n="pl"  c="Plural"/>
```
Comment (jonorthwash) [inline apertium-eng-cat.eng-cat.dix:84]: Why the inconsistent spacing?

File: apertium-eng-cat.eng-cat.dix [hunk 9]
```diff
@@ -11,99 +11,116 @@
...
+    <sdef n="nt"  c="Neuter"/>
+    <sdef n="GD"  c="Gender to be determined"/> 
+<!--Count/Mass-->
+    <sdef n="unc"  c="Uncountable (mass)"/>
+<!--Animacy-->
+    <sdef n="nn" c="Inanimate"/> 
+    <sdef n="an" c="Animate or inanimate"/> 
+    <sdef n="aa" c="Animate"/> 
+<!--Adjectives-->
+<sdef n="pred"  c="Predicative"/>
```
Comment (jonorthwash) [inline apertium-eng-cat.eng-cat.dix:53]: Why not indented?

File: apertium-eng-cat.eng-cat.dix (hunk 3 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:121]: I did so becuase 'pron' could not be found anywhere on the list of symbols page

Comment (jonorthwash) [reply apertium-eng-cat.eng-cat.dix:121]: It means "pronoun"

Comment (jonorthwash) [reply apertium-eng-cat.eng-cat.dix:121]: Should probably be moved to where it goes.

File: apertium-eng-cat.eng-cat.dix (hunk 9 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:53]: Indentation fixed

File: apertium-eng-cat.eng-cat.dix (hunk 8 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:84]: Spacing errors fixed

File: apertium-eng-cat.eng-cat.dix (hunk 7 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:86]: trailing spaces deleted

File: apertium-eng-cat.eng-cat.dix (hunk 6 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:72]: trailing spaces deleted

File: apertium-eng-cat.eng-cat.dix (hunk 5 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:41]: Extra spaces deleted

**Comment by gs-chaitanya** [comment]
Johnathan Washnigton please review

File: apertium-eng-cat.eng-cat.dix (hunk 3 above)
Comment (gs-chaitanya) [reply apertium-eng-cat.eng-cat.dix:121]: I have moved it to POS category

**Review by mr-martian — APPROVED** [review]

**Pull request merged**